src/components/StoryView/index.ts

```diff
@@ -1,4 +1,6 @@
-import StoryContainer from './StoryContainer';
+import { default as MultiStoryViewContainer } from './StoryContainer';
+import { default as StoryContainer } from './Story';
```

Comment:
@sagar-simformsolutions Here we can export Story as StoryCotainer and no need to export SingleStoryView, Ideally we want to enable play / pause in StoryContainer based on the extended prop